From cac66621b23029e0dbab0cbcf8720e602e92eafd Mon Sep 17 00:00:00 2001
From: Ronny Chevalier <chevalier.ronny@gmail.com>
Date: Thu, 25 Sep 2014 22:46:37 +0200
Subject: [PATCH] remove references of readahead

(cherry picked from commit cb607ecb84b3cb7299438ca6f7fab705b0a6de45)
---
 .gitignore | 1 -
 README     | 1 -
 TODO       | 7 -------
 3 files changed, 9 deletions(-)

diff --git a/.gitignore b/.gitignore
index 0b71f0973b..14f169135d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -101,7 +101,6 @@
 /systemd-quotacheck
 /systemd-random-seed
 /systemd-rc-local-generator
-/systemd-readahead
 /systemd-remount-api-vfs
 /systemd-remount-fs
 /systemd-reply-password
diff --git a/README b/README
index 99b66a8396..144036717c 100644
--- a/README
+++ b/README
@@ -30,7 +30,6 @@ AUTHOR:
 
 LICENSE:
         LGPLv2.1+ for all code
-        - except sd-readahead.[ch] which is MIT
         - except src/shared/MurmurHash2.c which is Public Domain
         - except src/shared/siphash24.c which is CC0 Public Domain
         - except src/journal/lookup3.c which is Public Domain
diff --git a/TODO b/TODO
index b07d664715..abe89b78a0 100644
--- a/TODO
+++ b/TODO
@@ -646,13 +646,6 @@ Features:
 
 * and a dbus call to generate target from current state
 
-* readahead:
-  - drop /.readahead on bigger upgrades with yum
-  - move readahead files into /var (look for them with .path units?)
-  - readahead: use BTRFS_IOC_DEFRAG_RANGE instead of BTRFS_IOC_DEFRAG ioctl, with START_IO
-  - readahead: when bumping /sys readahead variable save mtime and compare later to detect changes
-  - readahead: make use of EXT4_IOC_MOVE_EXT, as used by http://e4rat.sourceforge.net/
-
 * GC unreferenced jobs (such as .device jobs)
 
 * write blog stories about:
